### PR TITLE
Update the README and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Thomas-Boi**
This PR contains minor updates to the `README` and `CONTRIBUTING`. It's mostly:

- Explaining new standards (e.g. no need to optimize files)
- Explaining new features (e.g. how to update an icon)
- Clarification on existing features (e.g. `develop` vs `master`)
- Minor fixes (e.g. missing entry in the table of contents)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -326,6 +329,26 @@ As an example, let's assume you have created the SVGs for Redhat and Amazon Web
...
+<h2 id='updatingIcons'>Updating an Icon</h2>
+<p>
+  Sometimes, a company will update their logo or someone spotted an error in the SVG/icon that needs to be fixed. This means the current icon in our repository might need an update. The steps to do this is simple:
+</p>
+<ol>
+  <li>
+    Create a new commit to fix the SVGs.
+  </li>
+  <li>
+    Open a pull request to `develop` branch.
```
Comment (Panquesito7) [inline CONTRIBUTING.md:342]:
```suggestion
    Open a pull request based on the `develop` branch. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -461,7 +477,7 @@ We are running a Discord server. You can go here to talk, discuss, and more with
         Take the PRs/commits as a guideline. It's also a good idea to mention and thank all contributions who participated in the release (take description of <code><a href="https://github.com/devicons/devicon/pull/504">#504</a></code> as an example).
         </li>
         <li>
-        Rather than doing it manually, you can instead run <code>python ./.github/scripts/get_release_message.py $GITHUB_TOKEN</code> locally. Pass in your GitHub Personal Access Token for <code>$GITHUB_TOKEN</code> and you should see the messages. You can also use the `workflow_dispatch` trigger in the GitHub Actions tab.
+        We now have a script that will do this for us. Check the `build-bot`'s PR message in the last step. There should be a section where it displays the features that has been added in the release. You can copy the markdown there and use it for the release message.
```
Comment (Panquesito7) [inline CONTRIBUTING.md:480]:
```suggestion
        We now have a script that will do this for us. Check the `build-bot`'s PR message in the last step. There should be a section where it displays the features that have been added to the release. You can copy the markdown there and use it for the release message.
```

File: README.md [hunk 3]
```diff
@@ -159,19 +161,44 @@ Add css rules in your stylesheet
...
+        No icons for the latest SVGs. These will be build at every release.
+    </li>
+    <li>
+        Experimental changes.
+    </li>
+</ul>
+<b><code>master</code> contains:</b>
+<ul>
+    <li>
+        Latest official release, which contains the release' SVGs and icons.
```
Comment (Panquesito7) [inline README.md:192]:
```suggestion
        Latest official release, which contains the SVGs and icons.
```

**Review by Panquesito7 — COMMENTED** [review]
Good work! Just some minor comments. 🙂

**Review by amacado — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your help in all of this, @Thomas-Boi! 😄👍🎉

**Pull request merged**